metrics/rl_reliability/app.py

```diff
@@ -0,0 +1,6 @@
+import evaluate
+from evaluate.utils import launch_gradio_widget
+
+
+module = evaluate.load("lvwerra/rl_reliability", "online")
```

Comment:
we aren't going to host this within one of the `evaluate` orgs?

Reply:
That's a leftover from testing :) thanks for spotting.